dicom-archive/DICOM/DICOM.pm

```diff
@@ -266,57 +441,37 @@ sub sortByField {
   }
 }
 
-# Doesn't do anything in non-graphical case.
-sub loop {}
 
-1;
-__END__
+=pod
 
-=head1 NAME
-
-DICOM.pm is a Perl library that allows Perl programs to read the
-headers of medical image files conforming to DICOM standards.
+=head3 loop()
 
-=head1 SYNOPSIS
+Doesn't do anything in non-graphical case.
 
-  use DICOM;
-  my $dicom = DICOM->new();
-  $dicom->fill($dicomFile);
-  my $patientName = $dicom->value('0010', '0010');
-  
-=head1 DESCRIPTION
-
-DICOM (Digital Imaging and Communications in Medicine) is a standard
-designed to allow medical image files to be transferred, stored and
-viewed on different makes of computers. Perl is a multiplatform
-language that excels at system tasks such as file manipulation. It's
-easy to learn, particularly if you are familiar with C or the Unix
-shells and utility programs.
-
-This library provides the methods to read and parse a DICOM file, then
-to recover the contents of each header element by their standard DICOM
-group and element codes. Header element values can be edited (either
-through the GUI or command line) and the modified file written back to
-disk.
-
-=head2 Methods
-
-=over 4
+=cut
 
-=item * $object->fill($filename)
```

Comment:
this `loop` function got me confused!
is this `item * object ...` line not needed?

Reply:
I don't know. I don't understand the line. What do you think?